app/controllers/application_controller.rb

```diff
@@ -2,4 +2,23 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_filter :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
You could turn the 3 calls to `devise_parameter_sanitizer ` into 1 by looping over `[:sign_up, :account_update, :sign_in]`